### ISSUE TITLE
Receiving a new trade offer not working?

### ISSUE THREAD

**Issue by Fusionn** [Jessecar96/SteamBot#976]
I am using the default TradeOfferUserHander.cs and just trying to decline the offer instantly when the it receives a trade offer, in the log the notification is logged but nothing else is done, trade offer is not canceled and nothing is being logged. All I added was

```C#
Log.Success("TRADE OFFER RECEIVED IN TRADEOFFERUSERHANDLER.CS");
offer.Decline();
```

Kinda confused and would love some help.

If needed pastebins: 
http://pastebin.com/aVFbiiQh - TradeOfferUserHandler.cs
http://pastebin.com/GjVs4bDu - Bot.cs
http://pastebin.com/2JdsneJK - settings.json

**Comment by scholtzm** [comment]
Are you using API key bound to that particular account?

**Comment by Fusionn** [comment]
I got it to work using SteamTradeOfferBot.

**Comment by BlueRaja** [comment]
Does anyone use `offer.Decline()`?  Can we verify that it actually _does_ work?

IIRC I've had some issues with this in my bot, but I never investigated to check if it's an issue with `offer.Decline()` or something else..

**Comment by Fusionn** [comment]
@BlueRaja check out https://github.com/waylaidwanderer/SteamTradeOffersBot/tree/SteamTradeOffersBot/SteamBot works almost out of the gate(minus setup)

**Comment by BlueRaja** [comment]
Fixed in #981
